### ISSUE TITLE
issue with capturing long text-shadow and box-shadow

### ISSUE THREAD

**Issue by aminkodaganur** [niklasvh/html2canvas#763]
Hi Niklasvh,

First I really wanna thank you for such wonderful plugin...
I am actually developing a open source plugin for community wer i have one module the user can capture the screen but while capturing its not capturing a text-shadow and box-shadow ... actually am using long shadow here wer in i have multiple values for text-shadow and box-shadow 
E.g :  

```
  text-shadow: 0px 0px 0px rgb(1, 107, 179) ,1px 1px 0px rgb(1, 107, 179) ,2px 2px 0px rgb(1, 107, 179) ,3px 3px 0px rgb(1, 107, 179) ,4px 4px 0px rgb(1, 107, 179) ,5px 5px 0px rgb(2, 108, 180) ,6px 6px 0px rgb(2, 108, 180) ,7px 7px 0px rgb(2, 108, 180) ,8px 8px 0px rgb(2, 108, 180) ,9px 9px 0px rgb(3, 109, 181) ..........
```

so this is not getting captured ... And also some crazy stuff happening wen i give border-radius property..

here is a JSFiddel you can have a look https://jsfiddle.net/aminkodaganur/pn7pLhp6/2/ 

if possible please fix this issue I am eagerly waiting for this bcuz only this issue is stopping me to upload my project to github...

Thank you in advance ...

**Comment by baylock** [comment]
Same here: long text-shadows properties just don't work.

This will work:
`text-shadow: 2px 2px 2px #e0dedb;`
This will not:
`text-shadow: -2px 0 #e0dedb, 0 2px #e0dedb, 2px 0 #e0dedb, 0 -2px #e0dedb;`

Unfortunately, it doesn't give the same results.

**Comment by niklasvh** [comment]
Fixed in 1.0.0 https://jsfiddle.net/pn7pLhp6/3/
